libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/route/RouteEngine.java

```diff
@@ -52,7 +53,7 @@ public void fetchRoute(Point origin, RouteProgress routeProgress) {
       .routeOptions(currentOptions);
 
     // Add waypoints with the remaining coordinate values
```

Comment:
What about fixing `waypoints` https://github.com/mapbox/mapbox-navigation-android/blob/cb1af2fdd718cdc8979279616d93b4a739d2fea3/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/route/RouteEngine.java#L39 name into something like `remaining` and remove both comments (here and L38)?